test/testdata/integration-test.json

```
@@ -40,7 +40,7 @@
         {
           "id": "field2",
           "type": "BOOLEAN",
-          "value": "default",
+          "value": "true",
```

Related to my previous comment, one could maybe also set as a valid JSON:
```suggestion
          "value": true,
```

---

I've changed this in commit [e09eea8](https://github.com/leanix/vsm-discovery-publish-action/pull/36/commits/e09eea8afa0dc784423d77875f8b4365ae7207f1). This will only allow to use proper types in the json (e. g. `"value": 123` instead of `"value": "123"`).

---

We could also refine our schema later on to conditionally change the type of the `value` property, depending on the field type.